core/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestTags.scala

```diff
@@ -9,4 +9,8 @@ object LeonardoTestTags {
   // intent: do not run in given execution context
   // this is also synonmous with 'azure test'
   object ExcludeFromJenkins extends Tag("ExcludeFromJenkins")
+
+  // intent: run in cron scheduled tests only, not at every PR commit
```

Comment:
Is this still true now that we're running on merge to develop?

Reply:
Thanks for the catch, I will update the comment 👍